contracts/Voter.sol

```diff
@@ -220,6 +223,8 @@ contract Voter is IVoter, OwnableUpgradeable, ReentrancyGuardUpgradeable {
         if (lastVoted[_tokenId] > _epochTimestamp()) weightsPerEpoch[_time][_target] -= _votes;
```

Comment:
for `totalVotesForGaugePerEpoch` and `votesForGaugePerEpochPerOwner` the `_time` should ` =  _epochTimestamp() + TWO_WEEKS`